assets/chezmoi.io/docs/reference/commands/upgrade.md

```diff
@@ -17,14 +17,20 @@ requests should be sufficient for most cases.
     If you installed chezmoi using a package manager, the `upgrade` command
     might have been removed by the package maintainer.
 
-## `--executable` *filename*
+## Flags
+
+### `--executable` *filename*
 
 Set name of executable to replace.
 
-## `--method` *method*
+### `--method` *method*
 
 Set upgrade method.
 
+!!! danger
+
+    This flag is not safe for most people.
```

Comment:
I would prefer:

```
This flag should only be used when recommended by chezmoi developers.
```

I’d also add to the `Set upgrade method` to be something like:

```
Override upgrade method as determined by chezmoi itself.
```